File: work/nh/projects/nh_it_apps/client/src/app/models/IP.ts
export class IP {
    id:number = 0;
    networkInterfaceId : number = 0;
    ipAddress: string = "";
    subnet: string = "";
    gateway: string = "";
    dhcp: boolean = false;
    dhcpReserved: boolean = false;
    port: number = 0;
    comments: string = "";
    lastUpdatedBy: string = "";
    lastUpdated: Date = new Date();
}

File: work/nh/projects/nh_it_apps/api/Models/Entities/IP.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models.Entities
{
    public class IP : BaseEntity
    {
        public int NetworkInterfaceId { get; set; }

        [Column(TypeName="varchar(250)")]
        public string IPAddress { get; set; }

        [Column(TypeName="varchar(20)")]
        public string Subnet { get; set; }
        
        [Column(TypeName="varchar(250)")]
        public string Gateway { get; set; }
        
        public bool Dhcp { get; set; }
        public bool DhcpReserved { get; set; }
        public int Port { get; set; }

    }
}
